packages/cli/src/cmds/weakSubjectivityState.ts

```diff
@@ -0,0 +1,8 @@
+// TODO put the real server names/IPs once we have them
+export enum WeakSubjectivityServers {
+  mainnet = "http://localhost:8081",
+  prater = "http://localhost:8081",
+  pyrmont = "http://localhost:8081",
+  dev = "http://localhost:8081",
+  oonoonba = "http://localhost:8081",
```

Comment:
What are this servers / services? What software are they expected to run? Until this URLs are changed for something reachable publicly the feature won't work.

Reply:
These would be addresses for servers that would be running this code (assuming we decide to go with this scheme):
ChainSafe/eth2-ws-provider#1

> Until this URLs are changed for something reachable publicly the feature won't work.
That's what I put the `TODO` at the top of this list.

Reply:
I'll add a note in the description to say that we shouldn't merge until we arrive at consensus for the server code.

Reply:
These are now Infura endpoints instead, as per our discussion.